assets/bower.json

```diff
@@ -56,6 +56,11 @@
     "font-awesome": "4.6.1"
   },
   "overrides": {
+    "angular" : {
+      "dependencies" : {
+        "jquery": "2.1.4"
```

Comment:
This is a bit of a "gotcha".  Angular doesn't depend on jQuery, we just use it in place of `$jqLite`.  I had to override to force a dependency else wiredeps will load jQuery after Angular (instead of before), and if that happens it reverts `element` to `$jqLite` (and lots of stuff breaks).

TL;DR
Its not to resolve a version, its because wiredeps needs to think Angular depends on jQuery :smile: